docs/content/preview/yugabyte-platform/prepare/server-nodes-software/software-on-prem-manual.md

```diff
@@ -142,6 +142,15 @@ Physical nodes (or cloud instances) are installed with a standard AlmaLinux 8 se
 
     Ensure that the `yugabyte` user has permissions to SSH into the YugabyteDB nodes (as defined in `/etc/ssh/sshd_config`).
 
+    If you are using a CIS hardened image and want SSH access to database nodes, you need to manually add the `yugabyte` user to `sshd_config`.
+
+    ```sh
+    add_cmd = (
+            f"sudo sed -i '/^AllowUsers / s/$/ yugabyte/' /etc/ssh/sshd_config && "
```

Comment:
I think they have to run this on the node itself? Not sure where this add_cmd is used

Reply:
@shashwat-yb can you TAL

Reply:
This got copy pasted from itest code, get rid of the add_cmd we only want the actualy linux commands that are run directly on the host.